backend/oasst_backend/tree_manager.py

```diff
@@ -603,8 +605,34 @@ def check_condition_for_scoring_state(self, message_tree_id: UUID) -> bool:
                 return False
 
         self._enter_state(mts, message_tree_state.State.READY_FOR_SCORING)
+        self.update_message_ranks(rankings_by_message)
         return True
 
+    @managed_tx_method(CommitMode.COMMIT)
+    def update_message_ranks(self, rankings_by_message: Dict[int, int]) -> None:
+        for parent_msg_id, ranking in rankings_by_message.items():
+            unordered_ranking = []
+            for msg_reaction in ranking:
+                unordered_ranking.append(
+                    [msg_reaction.payload.payload.ranking, msg_reaction.payload.payload.ranked_message_ids]
```

Comment:
Why do we use the `ranking` field here, IMO it should be completely ignored and not be read/processed? Everything we need should be in `ranked_message_ids` .. i.e. the ids of the messages as they were ordered by the user.